### ISSUE TITLE
Sphinx Build fails with Extension error: cannot import name make_admonition

### ISSUE THREAD

**Issue by GadgetSteve** [wxWidgets/Phoenix#458]
Operating system: Win 10

wxPython version: 4.0.0b2

Stock or custom build: Source Build

Python version: 2.7.13 & 3.6.0

Stock or custom build: Stock

Sphinx v1.6.3

Description of the problem:
Running:
`python build.py sphinx`

Gives:

> Will build using: "C:\Python27\python.exe"                                                                                              
> 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)]                                                        
> Python's architecture is 32bit                                                                                                          
> cfg.VERSION: 4.0.0b2                                                                                                                    
>                                                                                                                                         
> Running command: sphinx                                                                                                                 
> sphinx-build -b html -d D:\Users\212303160\GitWorking\wxPhoenixSJB\docs\sphinx\build/doctrees . D:\Users\212303160\GitWorking\wxPhoenixS
> JB\docs\html                                                                                                                            
> Running Sphinx v1.6.3                                                                                                                   
> c:\python27\lib\site-packages\sphinx\util\compat.py:40: RemovedInSphinx17Warning: sphinx.util.compat.Directive is deprecated and will be
>  removed in Sphinx 1.7, please use docutils' instead.                                                                                   
>   RemovedInSphinx17Warning)                                                                                                             
>                                                                                                                                         
> Extension error:                                                                                                                        
> Could not import extension availability (exception: cannot import name make_admonition)                                                 
> Command 'sphinx-build -b html -d D:\Users\212303160\GitWorking\wxPhoenixSJB\docs\sphinx\build/doctrees . D:\Users\212303160\GitWorking\w
> xPhoenixSJB\docs\html' failed with exit code 1.                                                                                         
> Finished command: sphinx (14.441s)   

Searching for the missing import revealed: https://github.com/spinus/sphinxcontrib-images/issues/41

Tried the work around and it works.
```
diff --git a/docs/sphinx/conf.py b/docs/sphinx/conf.py
index a0675dfc..8b338c56 100644
--- a/docs/sphinx/conf.py
+++ b/docs/sphinx/conf.py
@@ -14,7 +14,9 @@

 import sys, os
 from datetime import datetime
-
+from docutils.parsers.rst.directives.admonitions import BaseAdmonition
+from sphinx.util import compat
+compat.make_admonition = BaseAdmonition
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.

```

**Comment by RobinD42** [comment]
I'll take a look when I can. In the meantime, Sphinx 1.5.5 is known to work

**Comment by mesalu** [comment]
I've been looking into this alognside #748, I think I know how to go about fixing this... but will need a bit of time to test it, and make sure the proper styles are still applied to the resulting admonition node.

Soon we will be free to upgrade sphinx! <*maniacal laughter*>

**Comment by RobinD42** [comment]
Fixed in #762
